includes/cub.h

```diff
@@ -104,6 +104,11 @@ typedef struct s_cub {
 	t_pair_i		*step;
 	int				hit;
 	int				side;
+	/* textures */
+	double			wall_x;
+	t_pair_i		*texel;
```

Comment:
what's texel sorry? 😅

Reply:
A texel is a pixel on a texture. It's important to differentiate this as texels need to be translated correctly into pixel values (stretching them or skipping some of the texels).

Reply:
oh so that's really a thing? could you add that explanation (just the texel = pixel on a texture part) to the comment?

Reply:
Done.